sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -56,6 +56,40 @@ static_assert(sizeof(uint64_t) == 8);
  */
 #define SUCCESS 0
 
+/**
+ * Builtin program error return values have the 31st bit set.  Programs
+ * may define their own values but their 31st bit should be unset
+ * to avoid conflicting with the builtin errors
+ */
+#define BUILTIN_ERROR_START 0x80000000
+
+/** Reserved */
+#define RESERVED_1 (BUILTIN_ERROR_START - 0)
```

Comment:
What's RESERVED_1 reserved for?  Same question for RESERVED_2

Reply:
Those are error values that don't make sense in C (like borrow failed).  I could omit them but would rather make it explicit that those values are hold meaning and should not be returned or redefined to mean something else in C

Reply:
Oh I see.   How about we use the same names in the C SDK to make it easier to align with the Rust SDK, but perhaps in the C SDK comment that this error is not applicable.    Just a very tiny nit